Formula/t/trafficserver.rb

```diff
@@ -17,69 +17,40 @@ class Trafficserver < Formula
     sha256 x86_64_linux:   "f5d0bd9a32ec12c38a583411b4a14d59c896db2aa599ad762baf86fac805c6c5"
   end
 
-  head do
-    url "https://github.com/apache/trafficserver.git", branch: "master"
+  depends_on "cmake" => :build
+  depends_on "ninja" => :build
+  depends_on "pkgconf" => :build
 
-    depends_on "autoconf" => :build
-    depends_on "automake" => :build
-    depends_on "libtool"  => :build
-  end
-
-  depends_on "pkg-config" => :build
+  depends_on "brotli"
   depends_on "hwloc"
-  depends_on macos: :mojave # `error: call to unavailable member function 'value': introduced in macOS 10.14`
+  depends_on "imagemagick"
+  depends_on "libmaxminddb"
+  depends_on "luajit"
+  depends_on "nuraft"
   depends_on "openssl@3"
   depends_on "pcre" # PCRE2 issue: https://github.com/apache/trafficserver/issues/8780
```

Comment:
should be pcre2?

Reply:
have tried but failed, cannot find pcre.